core/vm/jump_table.go

```diff
@@ -78,6 +79,13 @@ func validate(jt JumpTable) JumpTable {
 	return jt
 }
 
+func newBonehInstructionSet() JumpTable {
+	instructionSet := newMergeInstructionSet()
```

Comment:
this may not be true. These upgrades are for Ethereum, BSC does not enable yet: `Merge, London and Berlin`, we have to skip these sets.